features/admins/admin_post_issues.feature

```diff
@@ -14,6 +14,10 @@ Feature: Admin Actions to Post Known Issues
       And I press "Post"
     Then I should see "Known issue was successfully created"
       And I should see "First known problem"
+    When I follow "Admin Posts"
+      And I follow "Known Issues" within "#header"
```

Comment:
I think you can skip `When I follow "Admin Posts"`: it's not expanding the admin posts menu, just taking you to /admin_posts and then following the "Known Issues" link in the header.